pyEpiabm/pyEpiabm/toy_population_config.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+from .population import Population
+from .household import Household
+
+
+class ToyPopulationFactory:
+    """ Class that creates a toy population for use and returns in the simple
+    python model.
+    """
+    def make_pop(self, population_size: int, cell_number: int,
+                 microcell_number: int, household_number: int,
+                 if_households: bool = False):
+        """Method that initializes a population object with a given population size,
```

Comment:
```suggestion
        """Initialize a population object with a given population size,
```
They like docstrings to just be imperatives, or at least as brief as possible